### PR TITLE
feat: add a new plugin perplexity

### PR DESCRIPTION
**Pull request by LinkW77**
# Pull Request

## PR Description

A tool leveraging fine-tuned large language models like GPT-3.5 and GPT-4 in combination with Microsoft's Bing search engine to comprehensively answer all your questions.

## Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Performance enhancement
- [ ] Code refactor
- [ ] Documentation update
- [ ] Other, please describe:

## Checklist

Before submitting this PR, please make sure:

- [x] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) guidelines.
- [x] I have tested my changes locally to ensure they are effective.
- [x] I have updated the necessary documentation (if applicable).

### CODE REVIEW

File: plugin/bundles/perplexity/plugins/llama_3_sonar_large_32k_online/plugin.py
Comment (DynamesC) [inline plugin/bundles/perplexity/plugins/llama_3_sonar_large_32k_online/plugin.py:1]: Please rename the plugin to make it more readable

File: plugin/bundles/perplexity/plugins/llama_3_sonar_small_32k_online/plugin.py
Comment (DynamesC) [inline plugin/bundles/perplexity/plugins/llama_3_sonar_small_32k_online/plugin.py:1]: merge with the other plugin into one, and add an optional parameter for choosing models.

File: plugin/bundles/perplexity/plugins/get_answer_from_perplexity/plugin.py
Comment (DynamesC) [inline plugin/bundles/perplexity/plugins/get_answer_from_perplexity/plugin.py:1]: model should be extracted from input_params, rather than credentials

**Review by DynamesC — APPROVED** [review]
LGTM

**Review by taskingaijc — APPROVED** [review]
LGTM

**Pull request merged**